lib/definitions/constants.js

```diff
@@ -1,3 +1,10 @@
 export const HOME_URL = 'https://github.com/semantic-release/semantic-release';
 
 export const RELEASE_NAME = 'GitLab release';
+
+export const RETRY_CONFIGURATION = {
+    retry: {
+        limit: 3,
+        statusCodes: [429],
```

Comment:
I think we can go with the default here:

```suggestion
```

Reply:
I will close this PR. Thanks for reviewing!